srcpkgs/gnome-builder/template

```diff
@@ -13,11 +13,12 @@ makedepends="cairo-devel clang devhelp-devel enchant2-devel flatpak-devel
  libxml2-devel template-glib-devel vala-devel vte3-devel webkit2gtk-devel
  python3-gobject-devel sysprof-devel glade3-devel libportal-devel"
 depends="desktop-file-utils flatpak-builder python3-lxml devhelp python3-gobject"
+checkdepends="xvfb-run"
 short_desc="IDE for GNOME"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Apps/Builder"
-changelog="https://gitlab.gnome.org/GNOME/gnome-builder/raw/gnome-builder-3-36/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/gnome-builder/raw/master/NEWS"
```

Comment:
The correct one for this release branch would be `gnome-builder-3-40`, but I guess you prefer to stick with the one that documents major releases?